setup.py

```diff
@@ -41,6 +41,11 @@ def get_requirements():
     ],
     install_requires=get_requirements(),
     python_requires=">=2.6",
+    entry_points={
```

Comment:
This entry point looks correct, and the ones declared in the other projects shouldn't be needed.  If that doesn't work please let me know.